src/index.js

```diff
@@ -0,0 +1,72 @@
+import _, { indexOf } from 'lodash';
```

Comment:
The lodash package is unnecessary for this project, so please remove this `import`.